contracts/EtherCrowd.sol

```diff
@@ -57,11 +62,11 @@ contract EtherCrowd {
         string memory _videoUrl,
         string memory _thumbnailUrl,
         string memory _description,
-        uint _startDate,
+        //  no need to specify start date, it is the time at which the user calls the function
         uint _endDate
 
     ) external payable{
-        require(_endDate > _startDate, "End date has to be after start date");
```

Comment:
We must implement startDate, it is an essential feature.

Reply:
This line has been removed since last commit